baton.lua

```diff
@@ -96,33 +104,55 @@ end
 function Player:_updateControls()
   if not self._active then return false end
   for _, control in pairs(self._controls) do
-    control.value = 0
+    for i = 1, #control.values do
+      control.values[i] = 0
+    end
```

Comment:
Ironically, the fact that baton only iterates through the active device makes that whole discussion we had about an additional deadzone totally irrelevant. If you are using a keyboard, then it baton does not actually receive input from the joystick.